tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -1354,9 +1356,8 @@ trait WebActionsApiTests extends ControllerTestCommon with BeforeAndAfterEach wi
        *
        */
 
-      Post(s"$testRoutePath/$systemId/proxy/export_c.json", str) ~> addHeader(
-        "Content-type",
-        ContentTypes.`text/html(UTF-8)`.value) ~> Route.seal(routes(creds)) ~> check {
+      Post(s"$testRoutePath/$systemId/proxy/export_c.json", HttpEntity(ContentTypes.`text/html(UTF-8)`, str)) ~> Route
+        .seal(routes(creds)) ~> check {
```

Comment:
why do you need to set the content type explicitly here on the entity?

Reply:
Just maintaining the previous functionality as the content-type use to be set with `addHeader()`. If the content-type isn't set explicitly, the received content-type ends up being `text/plain; charset=UTF-8` instead of text/html; charset=UTF-8`.

Got rid of all the calls that didn't need the content-type set explicitly.